electron/constants/publishOptions.js

```diff
@@ -5,8 +5,7 @@ const publishOptions = {
   token: process.env.GH_TOKEN,
   private: false,
   publishAutoUpdate: true,
-  releaseType: process.env.IS_STAGING === 'true' ? 'prerelease' : 'release',
-  channel: process.env.IS_STAGING === 'true' ? 'alpha' : 'latest',
+  releaseType: process.env.IS_STAGING === 'true' ? 'draft' : 'release',
```

Comment:
no longer need this, we will always release to `draft` then promote to full release
```suggestion
  releaseType: 'draft',
```